documentation/player-portal/android/player-settings-android.md

```diff
@@ -0,0 +1,419 @@
+         * @return Player Settings
+         */
+        Settings setAdAutoPlayOnResume(boolean autoPlayOnResume);
+
+        /**
+         * Set the player buffers size
+         *
+         * @param loadControlBuffers LoadControlBuffers
+         * @return Player Settings
+         */
+        Settings setPlayerBuffers(LoadControlBuffers loadControlBuffers);
+
+        /**
+         * Set the Player's VR/360 support
+         *
+         * @param vrPlayerEnabled - If 360 media should be played on VR player or default player - default == true.
+         * @return - Player Settings.
+         */
+        Settings setVRPlayerEnabled(boolean vrPlayerEnabled);
+
+        /**
+         * Set the Player's preferredAudioTrackConfig.
+         *
+         * @param preferredAudioTrackConfig - AudioTrackConfig.
+         * @return - Player Settings.
+         */
+        Settings setPreferredAudioTrack(PKTrackConfig preferredAudioTrackConfig);
+
+        /**
+         * Set the Player's preferredTextTrackConfig.
+         *
+         * @param preferredTextTrackConfig - TextTrackConfig.
+         * @return - Player Settings.
+         */
+        Settings setPreferredTextTrack(PKTrackConfig preferredTextTrackConfig);
+
+        /**
+         * Set the Player's PreferredMediaFormat.
+         *
+         * @param preferredMediaFormat - PKMediaFormat.
+         * @return - Player Settings.
+         */
+        Settings setPreferredMediaFormat(PKMediaFormat preferredMediaFormat);
+
+        /**
+         * Set the Player's Subtitles
+         *
+         * @param subtitleStyleSettings - SubtitleStyleSettings
+         * @return - Player Settings
+         */
+        Settings setSubtitleStyle(SubtitleStyleSettings subtitleStyleSettings);
+    }  
+```
+
+Once you created a player instance you can set the above settings on it.
+
+## Example
+
+### Create a player:
+
+``` java
+ Player player = PlayKitManager.loadPlayer(context, pluginConfigs);
+```
+
+## Apply Player Settings if required:
+
+### Enable crossProtocolRedirect
+
+##### Example:
+
+```
+ //Configure if to player allow http/https mix.
+ player.getSettings().setAllowCrossProtocolRedirect(crossProtocolRedirectEnabled); // default is false
+```
+
+### Enable DRM Clear Lead Playback
+
+##### Example:
+
+```
+ //Configure if player will start playing clear lead in DRM content 
+ player.getSettings(). allowClearLead(true/false); // default is false
+```
+
+### Enable Secure Surface
+
+In case App wants to block ability to take screen capture
+
+##### Example:
+
+```
+ player.getSettings().setSecureSurface(isSurfaceSecured); // default is false
+```
+
+### Configure Player's PreferredMediaFormat.
+
+  By default if MediaEntry contains multiple sources from different formats player
+  will choose DASH (mpd) and if this format isavailable it will try to select hls and so on. 
+  in orer to force some other thay over the default formats priority use the following API
```

Comment:
orer.. should be order. isavailable, I'm guessing should be "is unavailable". No clue what "thay" is supposed to mean. Generally speaking, I'd rephrase this whole sentence.